### ISSUE TITLE
3D reconstruction pipeline based on light field image

### ISSUE THREAD

**Issue by LightFieldVision** [cdcseacave/openMVS#947]
Recently, I have been studying the modeling of light field image. I think light field image is very similar to multi-view image. Can openmvs be used for 3D reconstruction of light field image? thank you @cdcseacave

**Comment by 4CJ7T** [comment]
If the light field image is converted into an image format OpenMVS supports it should process fine after multi-view geometry.
If you want to use depth maps created from light field images then you'll have to write your own converter using Interface.h.

**Comment by cdcseacave** [comment]
no, I do not think OpenMVS is a good choice for processing light field images, for that a specialized approach should be use, like https://people.csail.mit.edu/changil/light-field-depth/

**Comment by cdcseacave** [comment]
Light field cameras are outside OpenMVS scope. See #1088.
